src/algebraic_geometry/prime_spectrum/basic.lean

```diff
@@ -520,6 +520,14 @@ begin
     exact zero_locus_anti_mono (set.singleton_subset_iff.mpr hfs) }
 end
 
+lemma is_basis_basic_opens :
```

Comment:
```suggestion
lemma is_basis_range_basic_open :
```

Reply:
Should I also rename the lemma immediately above?
```
lemma is_topological_basis_basic_opens : topological_space.is_topological_basis
(set.range (λ (r : R), (basic_open r : set (prime_spectrum R))))
```
or should I not rename this one, to be consistent?

Reply:
Hmm, let's leave it like it is then...